cpp/ps0000/ps0000_sl.cpp

```diff
@@ -1,15 +1,16 @@
-#include <iostream> // importing libraries should be started with # not @ and name is iostream only
+#include <iostream> // importing libraries should be started with # not @ and name is iostream only 2/2
 
-// it's main not Main
+// it's main not Main 1/1
 int main() { 
 
     // it's << not >> we are printing data to the console not getting data
-    // add std::endl; to end the line and add a new line (break)
+    // add std::endl; to end the line and add a new line (break) 4/4
     std::cout << "Hello, World!" << std::endl;
 
     // same message with escape \n
     std::cout << "Hello, World!\n";
```

Comment:
- [x] used `<<` for outputting instead of `>>`
- [x] used `\n` or `std::endl` in the right way
- [x] avoided syntax error by typing `;` at the end of the line
- [x] output is as required `Hello, World!` in a single line